README.md

```diff
@@ -103,9 +103,13 @@ in use when testing the ships functions, not during normal operations.
 
 Tasks for challenge completion:
 
-1. Refactor the code in a way that will make implementing the above requirements as easy and
-   straightforward as possible. Make sure you get the exact same output from the program as you did
+1. Refactor the code in a way that will make implementing all of the above requirements (both new and original) as easy and
+   straightforward as possible. Make sure you get the same output from the program as you did
    prior to the refactor.
```

Comment:
Are you sure we can require same output? Implementing CrowsNest will most definitely change the output.